### DISCUSSION TITLE
SSO support for the web portal

### DISCUSSION THREAD

**Discussion by Gb-rl** [dojo-engineering/feedback#19] [Q&A] [answered]
Hi all,

I am researching Dojo's capabilities regarding Single Sign-On integration for my organisation. Specifically, I would like to know:

Does Dojo currently support SSO for organisational logins using SAML or OAuth 2, particularly with platforms like Azure or Okta?
If not, are there any known plans or discussions about adding SSO support in the near future?
Are there any third-party tools or integrations that can be used with Dojo to enable SSO functionality?
I have reviewed the Dojo Developer Docs but haven't found any references to SSO support. Any guidance would be greatly appreciated!

Thank you!

**Comment by mdave16** [comment, marked as answer]
Hey @Gb-rl, We don't currently support SSO for organisational logins, however if you are a current customer,  please can you reach out to your account manager and we can discuss your needs better.
